cardinal/ecs/world.go

```diff
@@ -592,7 +596,7 @@ func (w *World) getITx(id transaction.TypeID) transaction.ITransaction {
 }
 
 // Namespace returns the world's namespace.
-func (w *World) Namespace() string {
+func (w *World) NamespaceAsString() string {
```

Comment:
You should add a String() method to the Namespace instead of having a special accessor. Calls would then look like this:

w.Namespace().String()